client/api/index.js

```diff
@@ -166,16 +166,17 @@ export default class WCStripeAPI {
 	/**
 	 * Creates and confirms a setup intent.
 	 *
-	 * @param {string} paymentMethodId The id of the payment method.
+	 * @param {Object} paymentMethod The id of the payment method.
```

Comment:
```suggestion
	 * @param {Object} paymentMethod      Payment method data.
	 * @param {string} paymentMethod.id   The ID of the payment method.
	 * @param {string} paymentMethod.type The type of the payment method.
```

Minor: I just noticed this param changed but the description wasn't updated. This suggested change matches with [WP JS documentation standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/#functions).

Reply:
Thanks James!